src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableValueStore.kt

```diff
@@ -237,6 +240,8 @@ class VariableValueStore(
             .on(VARIABLE_VALUES.ID.eq(VARIABLE_LINK_VALUES.VARIABLE_VALUE_ID))
             .leftJoin(VARIABLE_SECTION_VALUES)
             .on(VARIABLE_VALUES.ID.eq(VARIABLE_SECTION_VALUES.VARIABLE_VALUE_ID))
+            .leftJoin(VARIABLES)
```

Comment:
This could be a regular join (there will never be a variable value without a variable).